stdlib/email/base64mime.pyi

```diff
@@ -0,0 +1,9 @@
+from typing import Callable
+
+def header_length(bytearray: bytes) -> int: ...
+def header_encode(header_bytes: str | bytes, charset: str = ...) -> str: ...
```

Comment:
Can apply to both `str` and `bytes`. Looks like it's mainly intended for `bytes` but has a specific catch to allow `str` so allow both in typing.
![image](https://user-images.githubusercontent.com/80432516/147124170-17ebdf68-1a13-4601-99f1-259cf0844e54.png)